servicesCore/requestCore_service.py

```diff
@@ -1,35 +1,219 @@
+ import asyncio
+
 from artifacts import *
 from config import config
 from servicesContracts.requestEthereum_service import RequestEthereumService
+import servicesExtensions
```

Comment:
Is there a reason you need to import all of `servicesExtensions`? Same for `servicesContracts` below.

Reply:
nope, should be an explicit import